app/services/partners/archive_family_children.rb

```diff
@@ -0,0 +1,30 @@
+# This service object is meant to archive a family and all of
+# its children
+module Partners
+  class ArchiveFamilyChildren
+    include ServiceObjectErrorsMixin
+
+    # rubocop:disable Rails::SkipsModelValidations
+    def initialize(family:)
+      @family = family
+    end
+
+    def call
```

Comment:
I'm actually not a big fan of the instance pattern here - this can just be a plain module-level function. It's easier to mock, for one thing.